stylesheets/_global.scss

```diff
@@ -609,6 +612,281 @@ input[type=text], input[type=search], textarea {
   }
 }
 
+.full-screen-flow {
+  z-index: 1000;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  font-family: roboto-light;
+
+  color: black;
+  a {
+    color: $signal-blue;
+  }
+  background: linear-gradient(
+    to bottom,
+    rgb(238,238,238) 0%, // (1 - 0.41) * 255 + 0.41 * 213
+    rgb(243,243,243) 12%, // (1 - 0.19) * 255 + 0.19 * 191
+    rgb(255,255,255) 27%,
+    rgb(255,255,255) 60%,
+    rgb(249,249,249) 85%, // (1 - 0.19) * 255 + 0.19 * 222
+    rgb(213,213,213) 100% // (1 - 0.27) * 255 + 0.27 * 98
+  );
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  font-size: 10pt;
```

Comment:
When do use `em` vs `px` vs `pt`?

Reply:
I believe that we should start moving towards `em` for everything we possibly can. For the fonts, `pt` because I came from a design that specified them that way. For many design elements, `px` because `em` results in weird numbers sometimes.

Do you have a preference?

Reply:
I don’t feel strongly about a particular choice, but I’d like to standardize. I’ve seen `px` and `em` used mostly, `pt` not so much.